src/App.vue

```diff
@@ -46,6 +46,7 @@
 
       <!-- Breadcrumb -->
       <Breadcrumb
+        class="namerequest-sbc-breadcrumb"
```

Comment:
I would have liked to use an id here but it overrides the sbc assigned id and styles.